pkg/scheduler/eventhandlers.go

```diff
@@ -511,7 +514,7 @@ func addAllEventHandlers(
 			gvr, _ := schema.ParseResourceArg(string(gvk))
 			dynInformer := dynInformerFactory.ForResource(*gvr).Informer()
 			if handlerRegistration, err = dynInformer.AddEventHandler(
-				buildEvtResHandler(at, gvk, strings.Title(gvr.Resource)),
+				buildEvtResHandler(at, gvk, cases.Title(language.Und, cases.NoLower).String(gvr.Resource)),
```

Comment:
`cases.NoLower` is required to match the behavior with `strings.Title`, but if you don't need it, please let me know.